ur_robot_driver/include/ur_robot_driver/ur/ur_driver.h

```diff
@@ -192,6 +202,7 @@ class UrDriver
   std::thread watchdog_thread_;
   bool reverse_interface_active_;
   uint32_t reverse_port_;
+  uint32_t script_sender_port_;
```

Comment:
This member is currently not being used, so I would rather not have it as a member.